src/profile-logic/gecko-profile-versioning.js

```diff
@@ -1090,5 +1090,214 @@ const _upgraders = {
+          },
+        ],
+      },
+      {
+        name: 'Text',
+        tableLabel: '{marker.name} — {marker.data.name}',
+        chartLabel: '{marker.name} — {marker.data.name}',
+        display: ['marker-chart', 'marker-table'],
+        data: [{ key: 'name', label: 'Details', format: 'string' }],
+      },
+      {
+        name: 'Log',
+        display: ['marker-table'],
+        tableLabel: '({marker.data.module}) {marker.data.name}',
+        data: [
+          { key: 'module', label: 'Module', format: 'string' },
+          { key: 'name', label: 'Name', format: 'string' },
+        ],
+      },
+      {
+        name: 'DOMEvent',
+        tooltipLabel: '{marker.data.eventType} — DOMEvent',
+        tableLabel: '{marker.data.eventType}',
+        chartLabel: '{marker.data.eventType}',
+        display: ['marker-chart', 'marker-table', 'timeline-overview'],
+        data: [
+          { key: 'latency', label: 'Latency', format: 'duration' },
+          // eventType is in the payload as well.
+        ],
+      },
+      {
+        // TODO - Note that this marker is a "tracing" marker currently.
+        // See issue #2749
+        name: 'Paint',
+        display: ['marker-chart', 'marker-table', 'timeline-overview'],
+        data: [{ key: 'category', label: 'Type', format: 'string' }],
+      },
+      {
+        // TODO - Note that this marker is a "tracing" marker currently.
+        // See issue #2749
+        name: 'Navigation',
+        display: ['marker-chart', 'marker-table', 'timeline-overview'],
+        data: [{ key: 'category', label: 'Type', format: 'string' }],
+      },
+      {
+        // TODO - Note that this marker is a "tracing" marker currently.
+        // See issue #2749
+        name: 'Layout',
+        display: ['marker-chart', 'marker-table', 'timeline-overview'],
+        data: [{ key: 'category', label: 'Type', format: 'string' }],
+      },
+      {
+        name: 'IPC',
+        tooltipLabel: 'IPC — {marker.data.niceDirection}',
+        tableLabel:
+          '{marker.name} — {marker.data.messageType} — {marker.data.niceDirection}',
+        chartLabel: '{marker.data.messageType}',
+        display: ['marker-chart', 'marker-table', 'timeline-ipc'],
+        data: [
+          { key: 'messageType', label: 'Type', format: 'string' },
+          { key: 'sync', label: 'Sync', format: 'string' },
+          { key: 'sendThreadName', label: 'From', format: 'string' },
+          { key: 'recvThreadName', label: 'To', format: 'string' },
+        ],
+      },
+      {
+        name: 'RefreshDriverTick',
+        display: ['marker-chart', 'marker-table', 'timeline-overview'],
+        data: [{ key: 'name', label: 'Tick Reasons', format: 'string' }],
+      },
+      {
+        // The schema is mostly handled with custom logic.
+        name: 'Network',
+        display: ['marker-table'],
+        data: [],
+      },
+    ];
+
+    for (const processes of geckoProfile.processes) {
+      // Keep these empty.
```

Comment:
nit: Would be also good to note that why we are keeping these empty. Something like saying because we already include the whole marker schema in the parent process and there is no sub process specific marker schema maybe?